### PR TITLE
DOR fixes 

### PR DESCRIPTION
**Pull request by UmangSapient**
Few changes have been done in below 14 files for DOR alert fix.
Please review.

src/main/java/org/jarvis4ops/bean/ActiveSprintDetails.java
 src/main/java/org/jarvis4ops/bean/ActiveSprintResponse.java
 src/main/java/org/jarvis4ops/bean/DorParameters.java
 src/main/java/org/jarvis4ops/bean/Fields.java
 src/main/java/org/jarvis4ops/configurations/Configurations.java
 src/main/java/org/jarvis4ops/configurations/JiraConstants.java
 src/main/java/org/jarvis4ops/configurations/SlackMessagingConstants.java
 src/main/java/org/jarvis4ops/controller/JiraController.java
 src/main/java/org/jarvis4ops/controller/SlachController.java
 src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java
 src/main/java/org/jarvis4ops/helper/JiraHelper.java
 src/main/resources/jiraConstants.properties
 src/main/resources/slackMessaging.properties

### CODE REVIEW

File: src/main/java/org/jarvis4ops/bean/ActiveSprintDetails.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package org.jarvis4ops.bean;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ActiveSprintDetails {
```
Comment (raror3) [inline src/main/java/org/jarvis4ops/bean/ActiveSprintDetails.java]: As its a bean - you can rename it to SprintDetailBean (will then be reusable with the name)

File: src/main/java/org/jarvis4ops/bean/ActiveSprintResponse.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package org.jarvis4ops.bean;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ActiveSprintResponse {
```
Comment (raror3) [inline src/main/java/org/jarvis4ops/bean/ActiveSprintResponse.java]: I think renaming it to JiraValuesResponseBean or like might make it more eligible for re-using.
what do you think?

File: src/main/java/org/jarvis4ops/bean/DorParameters.java [hunk 3]
```diff
@@ -81,11 +82,25 @@ public String getOverallStatus() {
 	public void setOverallStatus(String overallStatus) {
 		this.overallStatus = overallStatus;
 	}
+	
 	/* (non-Javadoc)
 	 * @see java.lang.Object#toString()
 	 */
 	@Override
 	public String toString() {
```
Comment (raror3) [inline src/main/java/org/jarvis4ops/bean/DorParameters.java]: You can remove this toString method overall - if you are not using it anywhere (a suggestion)

File: src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java [hunk 4]
```diff
@@ -7,8 +7,11 @@
 
 import org.jarvis4ops.bean.DorParameters;
 import org.jarvis4ops.bean.IssueDetails;
+import org.springframework.boot.autoconfigure.jdbc.metadata.TomcatDataSourcePoolMetadata;
 import org.springframework.stereotype.Component;
 
+import ch.qos.logback.core.net.SyslogOutputStream;
```
Comment (raror3) [inline src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java]: Any specific need to use this import statement please?

File: src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java [hunk 5]
```diff
@@ -7,8 +7,11 @@
 
 import org.jarvis4ops.bean.DorParameters;
 import org.jarvis4ops.bean.IssueDetails;
+import org.springframework.boot.autoconfigure.jdbc.metadata.TomcatDataSourcePoolMetadata;
```
Comment (raror3) [inline src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java]: Any specific need to use this import statement please?

**Review by raror3 — CHANGES_REQUESTED** [review]
Request you to please help action on the review comments - so that we can make the code more cleaner @UmangSapient

File: src/main/java/org/jarvis4ops/bean/ActiveSprintDetails.java (hunk 1 above)
Comment (UmangSapient) [reply src/main/java/org/jarvis4ops/bean/ActiveSprintDetails.java]: I have changed it's name to SprintDetailBean

File: src/main/java/org/jarvis4ops/bean/ActiveSprintResponse.java (hunk 2 above)
Comment (UmangSapient) [reply src/main/java/org/jarvis4ops/bean/ActiveSprintResponse.java]: Changed its name to JiraActiveSprintResponseBean

File: src/main/java/org/jarvis4ops/bean/DorParameters.java (hunk 3 above)
Comment (UmangSapient) [reply src/main/java/org/jarvis4ops/bean/DorParameters.java]: I have removed this unused toString method

File: src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java (hunk 5 above)
Comment (UmangSapient) [reply src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java]: Removed unused import.

File: src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java (hunk 4 above)
Comment (UmangSapient) [reply src/main/java/org/jarvis4ops/helper/DorDodIssuesHelper.java]: Removed unused import.

**Comment by UmangSapient** [comment]
@raror3 I have incorporated review comments , additionally removed few more unused imports . Please review.

**Review by raror3 — APPROVED** [review]

**Pull request merged**